src/content/guidelines/motion/basics.mdx

```diff
@@ -80,18 +81,16 @@ IBM Design System recognizes different moments in user's experience, and offers
 </div></Tab>
 </WebsiteTabs>
 
-### Productive
+### Productive motion
 
-Productive motion creates a sense of efficiency and responsiveness, while being subtle. Use Productive motion for moments when user needs to focus on tasks—micro-interactions such as button interaction and dropdown menus, revealing additional information upon user's request, or rendering data tables and visualizations.
+Productive motion creates a sense of subtle efficiency and responsiveness. Productive motion is appropriate for moments when the user needs to focus on completing tasks. Microinteractions in Carbon such as button states, dropdowns, revealing additional information, or rendering data tables and visualizations were all designed with productive motion.
```

Comment:
How about "Productive motion creates a sense of efficiency and responsiveness,  while remain low-key / remain out of the way." Or a better word then "subtle" to replace "remain out of the way"? "Subtle efficiency" doesn't seem to emphasize the point here.